frontend/appflowy_flutter/lib/plugins/database_view/widgets/card/card.dart

```diff
@@ -143,29 +140,27 @@ class _RowCardState<T> extends State<RowCard<T>> {
               ),
             );
           }
-          // desktop
+
           return AppFlowyPopover(
             controller: popoverController,
             triggerActions: PopoverTriggerFlags.none,
             constraints: BoxConstraints.loose(const Size(140, 200)),
-            margin: const EdgeInsets.all(6),
             direction: PopoverDirection.rightWithCenterAligned,
-            popupBuilder: (popoverContext) => _handlePopoverBuilder(
-              context,
-              popoverContext,
-            ),
+            popupBuilder: (popoverContext) {
+              return RowActions(
+                viewId: context.read<CardBloc>().viewId,
+                rowId: context.read<CardBloc>().rowMeta.id,
+                groupId: widget.groupId,
+              );
+            },
             child: RowCardContainer(
               buildAccessoryWhen: () => state.isEditing == false,
-              accessoryBuilder: (context) {
-                if (widget.styleConfiguration.showAccessory == false) {
-                  return [];
-                } else {
-                  return [
-                    _CardEditOption(rowNotifier: rowNotifier),
-                    CardMoreOption(),
-                  ];
-                }
-              },
+              accessories: widget.styleConfiguration.showAccessory
+                  ? [
+                      _CardEditOption(rowNotifier: rowNotifier),
+                      const CardMoreOption(),
+                    ]
+                  : [],
```

Comment:
My personal preference - But it has no actual difference, just leave as is if you want.
```suggestion
              accessories: [
                if (widget.styleConfiguration.showAccessory) ...[
                  _CardEditOption(rowNotifier: rowNotifier),
                  const CardMoreOption(),
                ],
              ],
```